js/form.js

```diff
@@ -294,6 +363,9 @@
     $('#constituency_info').append('<p class="mp">' + mpForConstituency.mp + '</p>');
     $('#constituency_info').append('<p class="party">' + mpForConstituency.party + '</p>');
     $('#constituency_info').append('<p class="signatures_count"><span class="data">' + numberWithCommas(count) + '</span> signatures</p>');
+    if (PetitionMap.is_weighted) {
```

Comment:
I think we can always include this, not just for when the colouring is by percentage.